tests/default.nf.test

```diff
@@ -148,7 +148,7 @@ nextflow_pipeline {
                 outdir        = "$outputDir"
                 contrasts_yml = null
                 contrasts     = "https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/mus_musculus/rnaseq_expression/SRP254919.contrasts.csv"
-                report_file   = "$projectDir/tests/quarto.qmd,$projectDir/assets/differentialabundance_report.qmd"
+                report_file   = "$projectDir/tests/quarto.qmd,$projectDir/assets/report_differential.qmd"
```

Comment:
Can this `$projectDir/tests/quarto.qmd` be removed or should it be changed to `$projectDir/assets/report_exploratory.qmd`?